mne/epochs.py

```diff
@@ -3106,3 +3106,32 @@ def average_movements(epochs, head_pos=None, orig_sfreq=None, picks=None,
     _remove_meg_projs(evoked)  # remove MEG projectors, they won't apply now
     logger.info('Created Evoked dataset from %s epochs' % (count,))
     return (evoked, mapping) if return_mapping else evoked
+
+
+def segment_raw(raw, segment_length=1., verbose=True):
```

Comment:
yes please make it private and don't set verbose to True by default. See how we use the verbose decorator elsewhere